source/tasks/Deploy/DeployV6/input-parameters.test.ts

```diff
@@ -0,0 +1,104 @@
+import { Logger } from "@octopusdeploy/api-client";
+import { TaskWrapper } from "tasks/Utils/taskInput";
+import { getInputParameters } from "./input-parameters";
+import * as tasks from "azure-pipelines-task-lib/task";
+
+export class MockTaskWrapper implements TaskWrapper {
```

Comment:
This needs to move to a common location, but I initially had a reference issue so I put it here for now